source/_statement.html.md

```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
+*   missing level 1 headings (users may not be able to accurately determine the structure of content on the page)
+*   our status incident page is missing the main heading and the status home page has an incorrect heading hierarchy – this fails [success criterion 1.3.1: info and relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)
+*   our status page ‘subscribe to updates’ link does not contain programmatically discernible link text – this fails [success criterion 1.3.1: info and relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)
+*   our status page subscribe to updates form fields do not contain an explicit label – this fails [success criterion 1.3.1: info and relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)
+*   our status page subscribe to updates forms do not indicate to users that they are expandable or collapsible – this fails [success criterion 4.1.2: name, role, value](https://www.w3.org/TR/WCAG21/#name-role-value)
+*   our status page subscribe to updates forms contain some text that does not meet the minimum colour contrast requirements – this fails [success criterion 1.4.3: contrast, minimum](https://www.w3.org/TR/WCAG21/#contrast-minimum)
+*   our status page subscribe to updates form close link (x) is not descriptive enough for some users to determine its function or purpose – this fails [success criterion 2.4.4: link purpose, in context](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)
+*   our status page subscribe to updates forms, when submitted with incorrect data, do not inform screen reader users of the error message when it becomes available – this fails [success criterion 4.1.3: status messages](https://www.w3.org/TR/WCAG21/#status-messages)
 
-* the 'subscribe to updates' forms contain some text that does not meet the minimum colour contrast requirements, which fails WCAG 2.1AA 1.4.3 success criterion (Contrast (Minimum))
+We plan to have these issues addressed by the current supplier, or move to an alternative supplier in 2022.
 
-* the 'subscribe to updates' form close link ('x') is not descriptive enough for some users to work out its function or purpose, which fails WCAG 2.1A 2.4.4 success criterion (Link Purpose (In Context))
+## How we tested this service
 
-* the 'subscribe to updates' forms do not inform screen reader users of the error message that appears after incorrect data has been submitted, which fails WCAG 2.1AA 4.1.3 success criterion (Status Messages)
+The website was tested in May 2021 by DAC (digital accessibility centre) using a combination of automated and manual tests, including:
 
-We plan to have these issues addressed by the current supplier, or move to an alternative supplier in 2021.
+* JAWS 18 screen reader with Chrome
+* NVDA screen reader with Firefox
+* Dragon naturally speaking 15 with IE 11
```

Comment:
I think this is a brand name, so it should be "Dragon NaturallySpeaking 15" 

And we should spell out acronyms, so "with Internet Explorer 11"